docs/reference/db-authorization/introduction.md

```diff
@@ -96,15 +96,15 @@ In the Webhook case, the HTTP response contains the roles/user information as HT
 
 ## HTTP Headers
 
+***Note that using admin API key on HTTP headers is highly insecure and should be used only within protected networks.***
```

Comment:
I'd suggest wrapping this in a `caution` or `danger` admonition block so that its highlighted by Docusaursus: https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Muuuch better, tnx @simonplend 
![image](https://user-images.githubusercontent.com/668050/194865771-25ea8946-26bb-4dfc-9482-023c9ffa2680.png)